docs/user/integration/jaeger/README.md

```diff
@@ -0,0 +1,192 @@
+    helm repo add jaegertracing https://jaegertracing.github.io/helm-charts
+    helm repo update
+    ```
+
+### Install Jaeger
+
+> **NOTE:** It is officially recommended to install Jaeger with the [Jaeger operator](https://github.com/jaegertracing/helm-charts/tree/main/charts/jaeger-operator). Because the operator requires a cert-manager to be installed, the following instructions use a plain Jaeger installation. However, the described installation is not meant to be used for production setups.
+
+Run the Helm upgrade command, which installs the chart if not present yet.
+
+```bash
+helm upgrade --install --create-namespace -n $K8S_NAMESPACE $HELM_JAEGER_RELEASE jaegertracing/jaeger -f https://raw.githubusercontent.com/kyma-project/examples/main/jaeger/values.yaml
+```
+
+You can either use the [`values.yaml`](./values.yaml) provided in this `jaeger` folder, which contains customized settings deviating from the default settings, or create your own `values.yaml` file.
+
+### Verify the installation
+
+Check if the `jaeger` Pod was successfully created in the Namespace and is in the `Running` state:
+
+```bash
+kubectl -n $K8S_NAMESPACE rollout status deploy $HELM_JAEGER_RELEASE
+```
+
+### Activate a TracePipeline
+
+To configure the Kyma trace collector with the deployed Jaeger instance as the backend. To create a new [TracePipeline](https://kyma-project.io/#/telemetry-manager/user/03-traces), execute the following command:
+
+```bash
+cat <<EOF | kubectl -n $K8S_NAMESPACE apply -f -
+apiVersion: telemetry.kyma-project.io/v1alpha1
+kind: TracePipeline
+metadata:
+  name: jaeger
+spec:
+  output:
+    otlp:
+      protocol: http
+      endpoint:
+        value: http://$HELM_JAEGER_RELEASE-collector.$K8S_NAMESPACE.svc.cluster.local:4318
+EOF
+```
+  
+### Activate Istio Tracing
+
+To [enable Istio](../../03-traces.md#step-2-enable-istio-tracing) to report span data, apply an Istio telemetry resource and set the sampling rate to 100%. This approach is not recommended for production.
+
+```bash
+cat <<EOF | kubectl apply -f -
+apiVersion: telemetry.istio.io/v1alpha1
+kind: Telemetry
+metadata:
+  name: tracing-default
+  namespace: istio-system
+spec:
+  tracing:
+  - providers:
+    - name: "kyma-traces"
+    randomSamplingPercentage: 100.00
+EOF
+```
+
+### Access Jaeger
+
+1. To access Jaeger using port forwarding, run:
+
+   ```bash
+   kubectl -n $K8S_NAMESPACE port-forward svc/$HELM_JAEGER_RELEASE-query 16686
+   ```
+
+2. Open the Jaeger UI in your browser under `http://localhost:16686`.
+
+### Deploy a workload and activate Kyma's TracePipeline feature
+
+To see distributed traces visualized in Jaeger, follow the instructions for the [OpenTelemetry Demo App](../opentelemetry-demo/README.md).
+
+## Advanced Topics
+
+### Integrate with Grafana
+
+Jaeger can be provided as a data source integrated into Grafana. For example, it can be part of a Grafana installation as described in the [Prometheus tutorial](./../prometheus/README.md).
```

Comment:
```suggestion
Jaeger can be provided as a data source integrated into Grafana. For example, it can be part of a Grafana installation as described in the [Prometheus tutorial](https://github.com/kyma-project/examples/tree/main/prometheus).
```

Reply:
No, not for now as this example has no relation to the telemetry module yet